pkg/graph/schema.resolvers.go

```diff
@@ -862,6 +863,42 @@ func (r *mutationResolver) RejectIntake(ctx context.Context, input model.RejectI
 	}, err
 }
 
+func (r *mutationResolver) SubmitIntake(ctx context.Context, input model.SubmitIntakeInput) (*model.UpdateSystemIntakePayload, error) {
+	intake, err := r.store.FetchSystemIntakeByID(ctx, input.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	actorEUAID := appcontext.Principal(ctx).ID()
+	actorInfo, err := r.service.FetchUserInfo(ctx, actorEUAID)
+	if err != nil {
+		return nil, err
+	}
+
+	err = r.service.SubmitIntake(
+		ctx,
+		intake,
```

Comment:
If the status will always get changed to `models.SystemIntakeStatusINTAKESUBMITTED`, I'm wondering if it makes sense to apply that value here rather than passing that status in from the client.

Reply:
Looking at the component, looks like only the ID is passed in from the FE.   If that is the case, where is the status set?

Reply:
The status gets set as a side-effect of creating the actions in `pkg/services/action.go:81`